### DISCUSSION TITLE
How to scale fontsize in a viewbox?

### DISCUSSION THREAD

**Discussion by jeff9571** [AvaloniaUI/Avalonia#15396] [Q&A]
I have a viewbox with fixed size, and the content in the viewbox with a larger size, and it makes the content in viewbox looked so tiny.I wonder is there a way to make the font inside the viewbox appear the same size as outside the viewbox？
The demo code below:
![image](https://github.com/AvaloniaUI/Avalonia/assets/35749706/ef9594d5-1c01-4cc8-8b01-22267da2c262)
The effect:
![image](https://github.com/AvaloniaUI/Avalonia/assets/35749706/794902bc-3e24-4be1-b5d3-3e273ac7211e)

**Comment by jeff9571** [comment]
If I add a tooltip to the content textblock, it looks like the same size as outside the viewbox
<TextBlock FontSize="18" Text="Text in view box" ToolTip.Tip="ToolTip"/>

**Reply by timunie** [reply]
sure, becaus ToolTip is not affected by the scaling. I suggest to use a more flexible layout instead of ViewBox, tbh.
